style.css

```diff
@@ -0,0 +1,72 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px;
+  background-color: white;
+}
+
+header a {
+  text-decoration: none;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+}
+
+h1 {
+  font-weight: 800;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  padding: 15px;
+  padding-top: 100px;
+  font-family: 'Poppins', sans-serif;
+}
+
+p {
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  padding: 15px;
+  font-family: 'Poppins', sans-serif;
+}
+
+h2 {
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  padding: 15px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.body {
+  background-image: url('images/header-shapes\ mobile.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 100vh;
+  background-color: white;
+  border-radius: 0 0 0 35px;
+}
```

Comment:
- [ ] Your page's background image is not matched with the Figma design. Please work on it. It can be the issue for the `height: 100vh`. Please work on it.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![y](https://user-images.githubusercontent.com/70483835/164943847-14ff2765-81db-43f6-80e4-6dbaa8f5ea05.png) | ![f](https://user-images.githubusercontent.com/70483835/164943832-0e561db8-00f0-458f-9ff9-fd81a4e05058.png)

Reply:
> * [ ]  Your page's background image is not matched with the Figma design. Please work on it. It can be the issue for the `height: 100vh`. Please work on it.
> 
> Your design	Figma Design
> ![y](https://user-images.githubusercontent.com/70483835/164943847-14ff2765-81db-43f6-80e4-6dbaa8f5ea05.png)	![f](https://user-images.githubusercontent.com/70483835/164943832-0e561db8-00f0-458f-9ff9-fd81a4e05058.png)

- [ ] Kindly work on it. :+1: